htdocs/takepos/pay.php

```diff
@@ -421,8 +632,18 @@ function ValidateSumup() {
 	$i = $i + 1;
 }
 
-$keyforsumupbank = "CASHDESK_ID_BANKACCOUNT_SUMUP".$_SESSION["takeposterminal"];
+if (!empty($conf->stripe->enabled) && !empty($conf->global->STRIPE_CARD_PRESENT)) {
```

Comment:
Same. Must be used only if stripe payment enabled for the current terminal $_SESSION["takeposterminal"]